### PR TITLE
Add logging

### PR DESCRIPTION
**Pull request by Srul1k**
Here is what log looks like:
logpic

PR will close epambrest/quiz#64

### CODE REVIEW

**Comment by isalzh** [comment]
> Here is what log looks like:
> logpic
> 
> PR will close #64



> Here is what log looks like:
> logpic
> 
> PR will close #64

Looks pretty nice, from my point.

File: src/Teams/Controllers/MultipleAnswerQuestionController.cs [hunk 1]
```diff
@@ -69,6 +80,9 @@ public IActionResult AddMultipleAnswerQuestion([FromBody] MultipleQuestionAddMod
         [HttpPut]
         public IActionResult EditMultipleAnswerQuestion([FromBody] MultipleQuestionEditModel multipleAnswersQuestionDTO)
         {
+            IEnumerable<string> answers = multipleAnswersQuestionDTO.QuestionAnswers.Select(a => a.AnswersText);
+            _logger.LogInformation($"QuestionText: {multipleAnswersQuestionDTO.QuestionText}|Answers: {String.Join(", ", answers)}");
```
Comment (isalzh) [inline src/Teams/Controllers/MultipleAnswerQuestionController.cs:84]: The line 84 is duplicated 69's. The candidate to be extracted as a separate method.

File: src/Teams/Controllers/MarkParserController.cs [hunk 2]
```diff
@@ -1,13 +1,22 @@
...
+
+        public MarkParserController(ILogger<MarkParserController> logger)
+        {
+            _logger = logger;
+        }
+
         [HttpPost]
         public JsonResult GetMarkdownQuestion(string question)
         {
+            _logger.LogInformation("");
```
Comment (Tiktack) [inline src/Teams/Controllers/MarkParserController.cs:19]: Why you decided to do not log question string?

File: src/Teams/Controllers/MultipleAnswerQuestionController.cs [hunk 3]
```diff
@@ -51,12 +58,16 @@ public IActionResult EditMultipleAnswerQuestion(Guid id)
...
         {
+            _logger.LogInformation("");
             return View("AddMultipleAnswerQuestion");
         }
 
         [HttpPost]
         public IActionResult AddMultipleAnswerQuestion([FromBody] MultipleQuestionAddModel multipleAnswersQuestionDTO)
         {
+            IEnumerable<string> answers = multipleAnswersQuestionDTO.QuestionAnswers.Select(a => a.AnswersText);
+            _logger.LogInformation($"QuestionText: {multipleAnswersQuestionDTO.QuestionText}|Answers: {String.Join(", ", answers)}");
```
Comment (Tiktack) [inline src/Teams/Controllers/MultipleAnswerQuestionController.cs:69]: usually by styles guides we use not String or Int32, we are using string and just int. So please change to string.Join

File: src/Teams/Controllers/MarkParserController.cs (hunk 2 above)
Comment (isalzh) [inline src/Teams/Controllers/MarkParserController.cs:19]: Why don't log question parameter here?

**Review by Tiktack — APPROVED** [review]

**Review by isalzh — COMMENTED** [review]
Generally, looks good for me. Could be merged after resolving comments.

File: src/Teams/Controllers/MultipleAnswerQuestionController.cs (hunk 3 above)
Comment (isalzh) [reply src/Teams/Controllers/MultipleAnswerQuestionController.cs:69]: good catch!

File: src/Teams/Controllers/MarkParserController.cs (hunk 2 above)
Comment (Srul1k) [reply src/Teams/Controllers/MarkParserController.cs:19]: In HTTP Post methods without binding model (like [FromBody]), parameters are automatically logged.
getMarkdown

It is possible thanks to [${aspnet-request-form}](https://github.com/NLog/NLog/wiki/AspNetRequest-Form-Layout-Renderer) layout renderer.

**Pull request merged**